nexus/external-api/src/lib.rs

```diff
@@ -74,6 +74,13 @@ const PUT_UPDATE_REPOSITORY_MAX_BYTES: usize = 4 * GIB;
         allow_other_tags = false,
         policy = EndpointTagPolicy::ExactlyOne,
         tags = {
+            "affinity" = {
+                description = "Affinity groups give control over instance placement.",
```

Comment:
should this tag maybe also mention anti-affinity, so that users looking for anti-affinity APIs also know they'll find them here?
```suggestion
                description = "Affinity and anti-affinity groups give control over instance placement.",
```

Reply:
Sounds good, updated!